readme.md

```diff
@@ -84,6 +84,8 @@ Returns `true` for any object that implements its own `.next()` and `.throw()` m
 
 ##### .generatorFunction(value)
 
+##### .asyncFunction(value)
```

Comment:
perhaps show a little sneak-peak example?

f.e
```typescript
is.asyncFunction(async () => {});
// => true

is.asyncFunction(() => {});
// => false
```

Seeing a little of what the method does without having to check the tests can help people save time.

Reply:
Examples added.